### PR TITLE
v4 - Dropup caret direction incorrect

### LINKED ISSUE #17144: v4 - Dropup caret is missing
http://v4-alpha.getbootstrap.com/components/button-dropdown/#dropup-variation

### PR DESCRIPTION
**Pull request by kkirsche**
Fix #17144

### CODE REVIEW

**Comment by kkirsche** [comment]
JSBin: http://jsbin.com/mecego/edit?html,css,output

File: scss/_dropdown.scss [hunk 1]
```diff
@@ -24,6 +24,15 @@
   }
 }
 
+.dropup {
+  .dropdown-toggle {
+    &:after {
```
Comment (cvrebert) [inline scss/_dropdown.scss]: `&::after` , per #17052

Comment (kkirsche) [inline scss/_dropdown.scss]: Fixed. Sorry about that. I wasn't sure if you wanted double or single as I hadn't heard about #17058 yet.

File: scss/_dropdown.scss [hunk 2]
```diff
@@ -24,6 +24,15 @@
   }
 }
 
+.dropup {
+  .dropdown-toggle {
+    &::after {
```
Comment (mdo) [inline scss/_dropdown.scss]: Single `:` please.

File: scss/_dropdown.scss [hunk 3]
```diff
@@ -24,6 +24,15 @@
...
+      border-bottom: .3em solid;
```
Comment (mdo) [inline scss/_dropdown.scss]: You'll want to use `$caret-width` here.

File: scss/_dropdown.scss (hunk 2 above)
Comment (kkirsche) [inline scss/_dropdown.scss]: Changed back to single colon...

File: scss/_dropdown.scss (hunk 3 above)
Comment (kkirsche) [inline scss/_dropdown.scss]: And using the `$caret-width` variable now

File: scss/_dropdown.scss [hunk 4]
```diff
@@ -24,6 +24,15 @@
   }
 }
 
+.dropup {
+  .dropdown-toggle {
+    &:after {
+      border-bottom: $caret-width solid;
+      border-top: none;
```
Comment (mdo) [inline scss/_dropdown.scss]: Just realized we can change this from `none` to `0` later.

**Pull request merged**